source/Calamari.Shared/Integration/Certificates/WindowsX509CertificateStore.cs

```diff
@@ -126,41 +137,12 @@ public void AddPrivateKeyAccessRules(string thumbprint, StoreLocation storeLocat
                 if (!certificate.HasPrivateKey())
                     throw new Exception("Certificate does not have a private-key");
 
-                AddPrivateKeyAccessRules(privateKeyAccessRules, certificate);
+                CryptoKeySecurityAccessRules.AddPrivateKeyAccessRules(privateKeyAccessRules, certificate);
 
                 store.Close();
             }
         }
 
-        public CryptoKeySecurity GetPrivateKeySecurity(string thumbprint, StoreLocation storeLocation, string storeName)
```

Comment:
Moved below to place together with other netfx only calls to be wrapped in the compiler directive.